scripts/jenkins/qa_openstack.sh

```diff
@@ -168,8 +168,10 @@ $zypper --gpg-auto-import-keys -n ref
 # deinstall some leftover crap from the cleanvm
 $zypper -n rm --force 'python-cheetah < 2.4'
 
-# deinstall cloud-init and dependencies
+# deinstall cloud-init and dependencies (but keep cloud-final systemd unit around)
+cp /usr/lib/systemd/system/cloud-final.service /tmp
 $zypper -n rm --force -u cloud-init
+cp /tmp/cloud-final.service /usr/lib/systemd/system/cloud-final.service
```

Comment:
This magic^Wworkaround feels strange and could lead to further issues in the future (just my feeling).

My recommendation would be (if cloud-init really needs to be removed):
- deamonize the cleanvm run, so that it will not be killed by systemd (maybe run in detached screen, like install-suse-cloud)
- or fix/update dependecies of cloud-init (so that it does not need to be uninstalled)